### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Kiki-Jiji** [ONSBigData/precon#35]
It would be useful to be able to view the docs for this project.

Currently, I think, you have to clone and build them yourself?

A solution would be to use GitHub pages to serve the docs as this works well with sphinx.

**Comment by mitches-got-glitches** [comment]
Yeah, thanks for opening this issue. I'll look at the benefits of GitHub pages vs ReadTheDocs, set up a GitHub Action to deploy and then get them deployed as a start!
